base/threadingconstructs.jl

```diff
@@ -99,46 +99,78 @@ end
 """
     Threads.@threads [schedule] for ... end
 
-A macro to parallelize a `for` loop to run with multiple threads. Splits the iteration
-space among multiple tasks and runs those tasks on threads according to a scheduling
-policy.
-A barrier is placed at the end of the loop which waits for all tasks to finish
-execution.
-
-The `schedule` argument can be used to request a particular scheduling policy.
-
-Except for `:static` scheduling, how the iterations are assigned to tasks, and how the tasks
-are assigned to the worker threads is undefined. The exact assignments can be different
-for each execution. The scheduling option is a hint. The loop body code (including any code
-transitively called from it) must not make assumptions about the distribution of iterations
-to tasks or the worker thread in which they are executed. The loop body for each iteration
-must be able to make forward progress independent of other iterations and be free from data
-races. As such, synchronizations across iterations may deadlock.
+A macro to execute a `for` loop in parallel. The iteration space is distributed to
+coarse-grained tasks. This policy can be specified by the `schedule` argument. The
+execution of the loop waits for the evaluation of all iterations.
+
+See also: [`@spawn`](@ref Threads.@spawn),
+`pmap` in [`Distributed`](@ref man-distributed), and
+`BLAS.set_num_threads` in [`LinearAlgebra`](@ref man-linalg).
```

Comment:
```suggestion
See also: [`@spawn`](@ref Threads.@spawn) and
`pmap` in [`Distributed`](@ref man-distributed).
```

Address https://github.com/JuliaLang/julia/pull/44168/files#r805863793